mmv1/third_party/terraform/tests/resource_sql_user_test.go

```diff
@@ -279,6 +279,43 @@ func testAccSqlUserDestroyProducer(t *testing.T) func(s *terraform.State) error
 	}
 }
 
+func TestAccSqlUser_mysqlPasswordPolicy(t *testing.T) {
+	// Multiple fine-grained resources
+	skipIfVcr(t)
+	t.Parallel()
+
+	instance := fmt.Sprintf("i-%d", randInt(t))
```

Comment:
Can you change this to "tf-test-i%d"

We are trying to standardize on the tf-test prefix so we can programmatically sweep resources that get left behind

Reply:
Done